app/screens/claim/SelectClaim.tsx

```diff
@@ -20,8 +20,11 @@ const SelectClaim = ({
   const [isEnabled, setIsEnabled] = useState(false);
   const toggleSwitch = () => {
     setIsEnabled(previousState => !previousState);
-    const convertBoolToString: string = String(isEnabled);
-    item.setValue(convertBoolToString)
+    if(item.value === "false") {
+      item.setValue("true");
+    } else {
+      item.setValue("false");
+    }
```

Comment:
Can this be done in a ternary?  Something like:

`if(item.value === "false") ? item.setValue("true") : item.setValue("false");

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Reply:
Done